crawl-ref/source/dat/des/branches/tar.des

```diff
@@ -627,6 +627,527 @@ MAP
+TAGS:    transparent
+DEPTH:   Tar
+ORIENT:  centre
+NSUBST:  D = 1=+, E = 1=+
+SHUFFLE: DE
+SUBST:   D = m, E = x, 0 = ..000998
+MAP
+ x........................
+xxx.....x.x.x.......x.x.x.
+ x..0..xxDxDxx..0..xxDxDxx
+xxx...xxx.x.xxx...xxx.x.xxx
+ x.....E.....E.....E.....x
+xxx.0.xxx.0.xxx.0.xxx.0.xxx
+ x.....E.....E.....E.....x
+xxx.x.xxx...xxx.x.xxx...xxx
+ xxDxDxx..0..xxDxDxx..0..x
+ .x.x.x.......x.x.x.....xxx
+ ........................x
+ENDMAP
+
+NAME:    nicolae_tar_combbox
+TAGS:    transparent
+DEPTH:   Tar
+: if you.depth() == 1 then
+SUBST:   { = }
+: elseif you.depth_fraction() < 1 then
+SUBST:   { = {}
+: end
+SHUFFLE: DEFHJ
+SUBST:   DE : x, F : m, H : xm, J : xm, 0 = .009, 9 = 998
+MAP
+  x x x x x x
+ xxxxxxxxxxxxx
+xxx.x.x.x.x.xxx
+ x.0...0...0.x
+xxx..D...D..xxx
+ x..DEx.xED..x
+xxx..F...F..xxx
+ x.0HJ.{.JH0.x
+xxx..F...F..xxx
+ x..DEFJFED..x
+xxx..D.H.D..xxx
+ x.0...0...0.x
+xxx.x.....x.xxx
+ xxxxx...xxxxx
+  x x  @  x x
+ENDMAP
+
+NAME:    nicolae_tar_combstack
+TAGS:    transparent
+DEPTH:   Tar
+SHUFFLE: DE, FH
+NSUBST:  D = + / x, F = + / x
+SUBST:   EH = m
+MAP
+ x x x x x x x x
+xxxxxxxxxxxxxxxxx
+.x.x.x.x.x.x.x.x.
+....0...9...0....
+.x.x.x.x.x.x.x.x.
+xxDxExDxExDxExDxx
+ x.x.x.x.x.x.x.x
+xx..9...8...9..xx
+ x.x.x.x.x.x.x.x
+xxFxHxFxHxFxHxFxx
+.x.x.x.x.x.x.x.x.
+....0...9...0....
+.x.x.x.x.x.x.x.x.
+xxxxxxxxxxxxxxxxx
+ x x x x x x x x
+ENDMAP
+
+# Sometimes putting all the right spells and objects in the tomb isn't enough
+# to get the pharaoh into the good afterlife. RIP
+NAME:    nicolae_tar_entombed_in_hell
+TAGS:    transparent patrolling
+DEPTH:   Tar, !Tar:$
+ORIENT:  centre
+KFEAT:   } = }
+KMONS:   } = royal mummy
+KMONS:   1 = mummy priest
+KMONS:   2 = guardian mummy / ushabti / death scarab band / ancient champion / \
+             emperor scorpion / sphinx
+SHUFFLE: DE/FH/JK
+SUBST:   M = 12, DH = +, EFJK = x
+MAP
+            xxxx
+          xxx..x
+        xxx..2.E
+      xxx..M...x
+    xxx.D2...x+x
+  xxx...x..xxx.x
+xxx...M.xxxx.2.x
+x...1...xx.....x
+x.}.1.M.F.M....H
+x...1...xx.....x
+xxx...M.xxxx.2.x
+  xxx...x..xxx.x
+    xxx.J2...x+x
+      xxx..M...x
+        xxx..2.K
+          xxx..x
+            xxxx
+ENDMAP
+
+NAME:    nicolae_tar_entranced
+TAGS:    transparent patrolling
+DEPTH:   Tar:2-
+ORIENT:  centre
+KMONS:   1 = eidolon
+KMONS:   2 = tainted leviathan
+SHUFFLE: 12
+SUBST:   1 = 111., 2 = 2.
+MAP
+     x@@@x
+    xx...xx
+  xxx..1..xxx
+ xx2.......2xx
+ x...........x
+xx...........xx
+@.............@
+@.1....{....1.@
+@.............@
+xx...........xx
+ x...........x
+ xx2.......2xx
+  xxx..1..xxx
+    xx...xx
+     x@@@x
+ENDMAP
+
+# In the Fifth Circle of Dante's Inferno, the wrathful are punished in the foul,
+# stinking waters of the river Styx. The actively wrathful fight on the surface,
+# while the sullen lie below, sulking eternally. With no Phlegyas to help them,
+# players will have to ford the noxious river themselves. It'll be fine.
+NAME:    nicolae_tar_fifth_circle
```

Comment:
Consider: as a rare variant placing noxious bog for the ford with `KFEAT`.

Reply:
I thought about using toxic bog here, but the last time I submitted toxic bog as a KFEAT in a vault (for Swamp) it got nixed, so I didn't try again. But if you're cool with it... <cackles madly>

Reply:
In swamp the bog can kill monsters for the player. In Tar this is still true, but Tar monsters are much nastier to try to lure to the bog.

Just like the exposed diamond obelisk, there's a bit more latitude to be really wild in the hells.